core/integration/proxy_test.go

```diff
@@ -215,17 +224,18 @@ http_access allow localhost
 			}).Sync(ctx)
 		require.NoError(t, err)
 
-		squidLogs, err := c.Container().From(alpineImage).
-			WithMountedCache("/var/log/squidaccess", squidLogsVolume).
-			WithExec([]string{"cat", "/var/log/squidaccess/access.log"}).
-			Stdout(ctx)
 		require.NoError(t, err)
 		for _, test := range tests {
 			test := test
 			if test.proxyLogTest != nil {
 				t.Run(test.name+"-proxy-logs", func(t *testing.T) {
 					t.Parallel()
-					test.proxyLogTest(t, c, squidLogs)
+					test.proxyLogTest(t, c, func(ctx context.Context) (string, error) {
+						return c.Container().From(alpineImage).
+							WithMountedCache("/var/log/squidaccess", squidLogsVolume).
+							WithExec([]string{"cat", "/var/log/squidaccess/access.log"}).
```

Comment:
Does this need a cache-buster for the `cat`?

Reply:
Ah yeah good point! Wouldn't be surprised if that was part/whole of the flake before, I'll keep these updates but add that in too